### DISCUSSION TITLE
v2.5.4

### DISCUSSION THREAD

**Discussion by mashb1t** [lllyasviel/Fooocus#3500] [General]
## What's Changed
* fix: add handling for default "None" value of default_ip_image_* by @mashb1t in https://github.com/mashb1t/Fooocus/pull/70
* fix: correctly validate default_inpaint_mask_sam_model by @mashb1t in https://github.com/mashb1t/Fooocus/pull/71
* fix: check all dirs instead of only the first one by @mashb1t in https://github.com/lllyasviel/Fooocus/pull/3495
* fix: yield enhance_input_image to correctly preview debug masks by @mashb1t in https://github.com/lllyasviel/Fooocus/pull/3497
* docs: change code ownership from mashb1t to lllyasviel, see https://github.com/lllyasviel/Fooocus/discussions/3504

**Full Changelog**: https://github.com/lllyasviel/Fooocus/compare/v2.5.3...v2.5.4

<hr /><em>This discussion was created from the release <a href='https://github.com/lllyasviel/Fooocus/releases/tag/v2.5.4'>v2.5.4</a>.</em>

**Comment by mashb1t** [comment]
=> this is also 99% the last of my releases. You can find more information here: https://github.com/lllyasviel/Fooocus/discussions/3504

**Comment by bratelniku-droid** [comment]
<img width="514" height="696" alt="Screenshot 2026-03-04 123348" src="https://github.com/user-attachments/assets/bb1c29f0-854e-4406-bcc3-d32291f8b1cd" />



It is written in the "readme" file that the Fooocus program should be started with "run.bat" file.
It is fascinating!!! Except of one small problem: there NO that f'''ing "run.bat" file inside of zip of the last version 2.5.5...

**Reply by mashb1t** [reply]
@bratelniku-droid There is, please check the latest activities in your antivirus software.

You can also create one with the content of https://github.com/lllyasviel/Fooocus/blob/ae05379cc97bc4361ec8b4ec90193dab21be763f/build_launcher.py#L9-L10
You may want to remove the placeholders and add the corresponding preset though.
